pkg/awsManager/vpcVpnManager.go

```diff
@@ -231,19 +233,16 @@ func DeleteNetworkLoadBalancer(client clientpkg.Client, vpcID *string) error {
 			break
 		}
 	}
-
 	return nil
 }
 
-func DetachAndDeleteNetworkInterface(client clientpkg.Client, vpcID *string) error {
+func detachAndDeleteNetworkInterface(client clientpkg.Client, vpcID *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file